src/changelog/2024/07/immersive-editor.md

```diff
@@ -13,7 +13,7 @@ We found out that many users have to frequently switch between the Node-RED Edit
 
 We're always seeking to reduce friction in the FlowFuse user experience, and as such, we've introduced a large overhaul of the developer experience for Node-RED when running in FlowFuse, in what we're calling the "Immersive Editor".
 
-![Node-RED Editor toolbar button for the assistant](/blog/2024/07/images/immersive-editor.png){data-zoomable}
+![Node-RED Editor toolbar button for the assistant](./blog/2024/07/images/immersive-editor.png){data-zoomable}
```

Comment:
did you test this locally, I'd expect this to not work, and look for a nested `/blog` folder within the `changelog/2024/07` folder?

Reply:
You're probably right, but I could see the error locally and with this change, it doesn't show any errors.

Reply:
From what I can tell, the image pipeline expects all images to be relative paths, not absolute. 

I've run a full build with this change in locally, with the image pipeline running - and it appears to have worked. That does surprise me slightly... but not going to sweat it if it unblocks the website build.

Reply:
The deploy preview also confirms this is working - the changelog entry is there with the expected image.

Reply:
The logical path would have been `../../../blog/2024/07/images/immersive-editor.png` right?